docs/source/nbconvert_library.ipynb

```diff
@@ -511,27 +511,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The following demonstration, as requested in [a GitHub issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
+    "The following demonstration, as requested in [a GitHub issue](https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
```

Comment:
I remember it having a README saying something like "this repo is deprecated". I don't remember ever deleting the old repo, though the PR merging it into IPython did preserve all the commit history. Old links to that repo don't resolve, which suggests that maybe it was deleted. I don't know how or why that would have happened, though.